content/zh/blog/2021/wasm-api-alpha/index.md

```diff
@@ -0,0 +1,188 @@
+---
+title: WebAssembly 插件 Alpha 版可用性公告
+description: 介绍新的 Wasm 插件 API 以及 Envoy 和 Istio 中基于 Wasm 的插件支持的更新。
+publishdate: 2021-12-16
+attribution: "Daniel Grimm (Red Hat), Pengyuan Bian (Google), Takeshi Yoneda (Tetrate); Translated By: Wilson Wu (DaoCloud)"
```

Comment:
```suggestion
attribution: "Daniel Grimm (Red Hat), Pengyuan Bian (Google), Takeshi Yoneda (Tetrate); Translated by Wilson Wu (DaoCloud)"
```
Not sure whether this form is agreed mostly, which can have some more variants:
- Wilson Wu (DaoCloud)
- Wilson Wu (DaoCloud) translated/localized
- localized by Wilson Wu (DaoCloud)

Reply:
For the clear format, I think Wilson Wu (DaoCloud) translated is a good way, still need more suggestions @ericvn @hanxiaop @kebe7jun

Reply:
`Translated by Wilson Wu (DaoCloud)` seems clear and unify.

Reply:
```suggestion
attribution: "Daniel Grimm (Red Hat), Pengyuan Bian (Google), Takeshi Yoneda (Tetrate); translated by Wilson Wu (DaoCloud)"
```
Same as @kebe7jun, however not sure if we need to capitalize these words after `;`

Reply:
chatGPT said both are correct 😄 :

```
attribution: "..., Takeshi Yoneda (Tetrate); translated by Wilson Wu (DaoCloud)"
attribution: "..., Takeshi Yoneda (Tetrate); Translated by Wilson Wu (DaoCloud)"
```

Reply:
So I will accept this: Translated by Wilson Wu (DaoCloud)